tests/utility/v2/test_component_analyses.py

```diff
@@ -283,10 +283,6 @@ def test_get_batch_ca_data_semver(self, _mockca):
         self.assertIsInstance(result, dict)
         self.assertIn('result', result)
         self.assertIsInstance(result.get('result'), dict)
-        self.assertIn('requestId', result)
```

Comment:
These fields are not used in bossiness logic, and concurrent data fetcher ignores while doing concat